setup.py

```diff
@@ -107,6 +107,19 @@ def get_build_env():
             **os.environ.copy(),
             "TT_METAL_HOME": Path(__file__).parent,
             "TT_METAL_ENV": "production",
+            # Need to create static lib for tt_metal runtime because currently
```

Comment:
This comment's information is correct, however I would you like to reword it. Please change it to the following:

```
"""
Currently, the ttnn (ttnn/_ttnn.so) and tt_lib (tt_lib/_C.so)
both link to the tt_metal runtime. The specific thing in
ttnn linking to tt_metal is likely the implementation of
ttnn.manage_device.

However, because of the singleton design of
tt_cluster in tt_metal, both tt_lib and ttnn will have a
copy of the cluster object, causing a hang during
device operations in ttnn, such as calling
output = ttnn.to_torch(output).

Ultimately, we will not statically build tt_metal, and
opt to dynamically set rpath of the bindings to the
packaged libs for now.
"""
```